Client/core/CScreenShot.cpp

```diff
@@ -35,6 +35,9 @@ static bool    ms_bIsSaving = false;
 static uint    ms_uiWidth = 0;
 static uint    ms_uiHeight = 0;
 
+// whether we want to actually save photo in documents folder
+static bool savePhotoInDocuments = false;
```

Comment:
Is there a specific reason for making it static? Isn't it better to make it a member of CScreenShot?

Reply:
I just followed the previous variable declarations. if you say its better, then I do it.

Reply:
It should be a member because it breaks a class invariant in the current state.

Reply:
when I put the var in private I get unresolved external symbol error

Reply:
You need to remove a `static` specifier as well.

Reply:
cannot make the variable non static as the functions using it are static. i dont want to rewrite the whole class just to make one variable non static.

Reply:
I see now. Sorry for misleading you. I didn't see that the whole CScreenShot class is static.